From f22250d44ace0dd9ca76832bbf99fc1ed3fad022 Mon Sep 17 00:00:00 2001
From: Sabyasachi Nayak <sabyasachi.nayak@windriver.com>
Date: Tue, 16 Jul 2024 02:36:01 -0400
Subject: [PATCH] Removing the selector.matchLabels to unblock upgrade

Signed-off-by: Sabyasachi Nayak <sabyasachi.nayak@windriver.com>
---
 templates/_helpers.tpl | 2 --
 1 file changed, 2 deletions(-)

diff --git a/templates/_helpers.tpl b/templates/_helpers.tpl
index ee6dbdc..7bf3a99 100644
--- a/templates/_helpers.tpl
+++ b/templates/_helpers.tpl
@@ -55,8 +55,6 @@ Selector labels
 {{- define "security-profiles-operator.selectorLabels" -}}
 app: security-profiles-operator
 name: security-profiles-operator
-app.kubernetes.io/name: {{ include "security-profiles-operator.name" . }}
-app.kubernetes.io/instance: {{ include "security-profiles-operator.name" . }}
 {{- end }}
 
 {{/*
-- 
2.25.1

