From c808b3c1f1c09122883e86957787600f2aecd6ee Mon Sep 17 00:00:00 2001
From: Md Irshad Sheikh <mdirshad.sheikh@windriver.com>
Date: Tue, 1 Jul 2025 05:40:23 -0400
Subject: [PATCH] skip processing of faulty app armor profile

---
 internal/pkg/nonrootenabler/nonrootenabler.go | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/internal/pkg/nonrootenabler/nonrootenabler.go b/internal/pkg/nonrootenabler/nonrootenabler.go
index a5a5816..86fa241 100644
--- a/internal/pkg/nonrootenabler/nonrootenabler.go
+++ b/internal/pkg/nonrootenabler/nonrootenabler.go
@@ -22,6 +22,7 @@ import (
 	"fmt"
 	"os"
 	"path"
+	"path/filepath"
 
 	"github.com/go-logr/logr"
 	"sigs.k8s.io/release-utils/util"
@@ -195,6 +196,12 @@ func (*defaultImpl) SaveKubeletConfig(filename string, kubeletConfig []byte, per
 }
 
 func (*defaultImpl) InstallApparmor(manager apparmorprofile.ProfileManager, filename string) error {
+	if filepath.Base(filename) == "spo-apparmor.yaml" {
+		// Skipping known problematic AppArmor profile
+		fmt.Printf("Skipping AppArmor profile installation for %s\n", filename)
+		return nil
+	}
+
 	content, err := os.ReadFile(filename)
 	if err != nil {
 		return fmt.Errorf("reading apparmor profile content: %w", err)
-- 
2.34.1

